x-pack/plugins/ml/public/application/aiops/explain_log_rate_spikes.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { ExplainLogRateSpikesSpec } from '@kbn/aiops-plugin/public';
+import { useMlKibana, useTimefilter } from '../contexts/kibana';
+import { HelpMenu } from '../components/help_menu';
+
+import { MlPageHeader } from '../components/page_header';
+
+export const ExplainLogRateSpikesPage: FC = () => {
+  useTimefilter({ timeRangeSelector: false, autoRefreshSelector: false });
+  const {
+    services: { docLinks, aiops },
+  } = useMlKibana();
+
+  const [ExplainLogRateSpikes, setExplainLogRateSpikes] = useState<ExplainLogRateSpikesSpec | null>(
+    null
+  );
+
+  useEffect(() => {
+    if (aiops !== undefined) {
+      const { getExplainLogRateSpikesComponent } = aiops;
+      getExplainLogRateSpikesComponent().then(setExplainLogRateSpikes);
+    }
+  }, []);
+
+  return ExplainLogRateSpikes ? (
```

Comment:
Why are we checking the existence of `ExplainLogRateSpikes` twice here?

Reply:
Oh great catch, I guess there must be another check in place for something else where I copied this from, removed the outer check. (I wonder if we can simplify this whole setup a bit but for now kept it mostly in sync with how it's set up in the `data_visualizer` plugin). Fixed in bbe2c5e.